src/core/public/application/capabilities/capabilities_service.tsx

```diff
@@ -88,4 +65,18 @@ export class CapabilitiesService {
       capabilities,
     };
   }
+
+  private async fetchCapabilities(http: HttpStart, appIds: string[]): Promise<Capabilities> {
+    const payload = JSON.stringify({
+      applications: appIds,
+    });
+
+    const url = http.anonymousPaths.isAnonymous(window.location.pathname)
+      ? '/api/core/capabilities/defaults'
+      : '/api/core/capabilities';
```

Comment:
btw is this public API?

Reply:
It is. Have been added by security team though, not platform.

https://github.com/elastic/kibana/blob/a149497b73d7046628c1c75490538173f11f4a09/src/core/public/http/types.ts#L22-L35

Reply:
I asked about endpoints. my bad 😅

Reply:
oh. Good question then. It was previously accessed from injected metadatas, so was not. My guess would be that we want it to stay that way, but any argument against that is welcome. What does that fundamentally change? documentation?

Reply:
Not much right now. In future: compatibility guarantee, documentation.
That's a topic for a broader discussion. I'd assume that all the platform API is not public.